### PR TITLE
Add simple_diarizer with submodules

### PR DESCRIPTION
**Pull request by wghezaiel**
- Add submodules of simple_diarizer in simple
- Install simple_diarizer from submodules

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[submodule "simple/simple_diarizer"]
+	path = simple/simple_diarizer
+	url = git@github.com:linto-ai/simple_diarizer.git
+	branch = NME_SC
+[submodule "simple/diarization/simple_diarizer"]
+	path = simple/diarization/simple_diarizer
+	url = git@github.com:linto-ai/simple_diarizer.git
+	branch = NME_SC
```
Comment (Jeronymous) [inline .gitmodules:8]: I think one of the two submodule is to be removed here

File: simple/diarization/processing/speakerdiarization.py [hunk 2]
```diff
@@ -2,9 +2,12 @@
 import logging
 import os
 import time
-import simple_diarizer.diarizer
 import memory_tempfile
 import werkzeug
+import simple_diarizer.diarizer
+
+
+
```
Comment (Jeronymous) [inline simple/diarization/processing/speakerdiarization.py:10]: Please remove extra new lines

(and it was unnecessary to change this file)

File: simple/requirements.txt [hunk 3]
```diff
@@ -7,4 +7,6 @@ gunicorn>=20.1.0
 memory-tempfile
 pyyaml>=5.4.1
 supervisor>=4.2.2
-git+https://github.com/linto-ai/simple_diarizer.git@02fbb851bb820b9e179ce3c84a12ec8b65d304e2
\ No newline at end of file
+torch>=1.9.0
+
+
```
Comment (Jeronymous) [inline simple/requirements.txt:12]: why torch>=1.9.0 was added?
shouldn't it be in the requirements of simple_diarizer only?

Alos, please remove extra new lines

File: .gitmodules (hunk 1 above)
Comment (wghezaiel) [reply .gitmodules]: Corrected

File: simple/diarization/processing/speakerdiarization.py (hunk 2 above)
Comment (wghezaiel) [reply simple/diarization/processing/speakerdiarization.py]: Corrected

File: simple/requirements.txt (hunk 3 above)
Comment (wghezaiel) [reply simple/requirements.txt]: it's for USE_GPU option

**Pull request closed without merging**